app/scripts/metamask-controller.js

```diff
@@ -2945,6 +2967,9 @@ export default class MetamaskController extends EventEmitter {
         this.networkController.getNetworkConfigurationByNetworkClientId.bind(
           this.networkController,
         ),
+      getNetworkClientById: this.networkController.getNetworkClientById.bind(
+        this.networkController,
+      ),
```

Comment:
Added API callback for `getNetworkClientById` method/action.

Reply:
I don't see this being used in this PR

Reply:
Yeah, I would say let's not add this for now if we don't need it. We can always add it later.

Reply:
```suggestion
```